bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/Ngs2Strategy.scala

```diff
@@ -162,12 +166,28 @@ final case class Ngs2Strategy(
         // that is repositioned at each science offset and then the intersection
         // of these will be the valid range.  For now though, we'll just use the
         // candidates within the (0, 1') range from GemsCandidates.
-
-        val cwfs =
-          GemsCandidates.groupAndValidate(ctx, posAngles(ctx), ts)
-            .foldLeft(List.empty[SiderealTarget]) { case (cwfs, gc) =>
-              gc.cwfsCandidates ++ cwfs
-            }
+        val pf                     = PatrolField.fromRadiusLimits(JAngle.ANGLE_0DEGREES, JAngle.arcmins(1.3))
```

Comment:
It will never change, but it might be good to explicitly add 18 arcsec to `CanopusWfs.RADIUS_ARCSEC` (making it public first) for clarity instead of using 1.3' directly.